packages/react-ui/src/widgets/legend/LegendRamp.js

```diff
@@ -70,25 +109,29 @@ const useStylesStepsDiscontinuous = makeStyles(() => ({
 
 function StepsDiscontinuous({ labels = [], palette = [], max, min }) {
```

Comment:
Same as in other components... do you think we should split into different files?

Reply:
Since StepsDiscontinuous is only used in LegendRamp and the file isn't too big, I would keep it here. But I can split it if you want, np about that.

Reply:
No need now, Sergio, but if you create new ones, I would suggest so, to always follow that rule one file per component